homeassistant/components/zwave_js/entity.py

```diff
@@ -67,12 +67,20 @@ def on_value_update(self) -> None:
         To be overridden by platforms needing this event.
         """
 
+    async def _async_poll_value(self, value_or_id: str | ZwaveValue) -> None:
+        """Poll a value."""
+        # We log an error instead of raising an exception because this service call occurs
+        # in a separate task and we don't want to raise the exception in that separate task
+        # because it is confusing to the user.
+        try:
+            await self.info.node.async_poll_value(value_or_id)
+        except BaseZwaveJSServerError as err:
+            LOGGER.error("Error while refreshing value %s: %s", value_or_id, err)
+
     async def async_poll_value(self, refresh_all_values: bool) -> None:
         """Poll a value."""
         if not refresh_all_values:
-            self.hass.async_create_task(
-                self.info.node.async_poll_value(self.info.primary_value)
-            )
+            self.hass.async_create_task(self._async_poll_value(self.info.primary_value))
```

Comment:
Do you remember why we create a task for this call instead of awaiting it?

Reply:
I don't... Perhaps the thought was that we don't want to block on battery devices, but the controller should handle that so I'm not sure why we felt the need to do so here.